src/plugins/displayLayout/components/BoxView.vue

```diff
@@ -21,15 +21,18 @@
  *****************************************************************************/
 
 <template>
-    <layout-frame :item="item"
-                  :grid-size="gridSize"
-                  @move="(gridDelta) => $emit('move', gridDelta)"
-                  @endMove="() => $emit('endMove')">
-        <div class="c-box-view"
-             :style="style">
-        </div>
-    </layout-frame>
- </template>
+<layout-frame
+    :item="item"
+    :grid-size="gridSize"
+    @move="(gridDelta) => $emit('move', gridDelta)"
+    @endMove="() => $emit('endMove')"
+>
+    <div
+        class="c-box-view"
+        :style="style"
+    />
```

Comment:
<div> can't be self closed.

Reply:
disabled "vue/html-self-closing" rule but for now I have not undone the autolinting that has self closed elements like above.

Reply:
@akhenry , @psarram , @deeptailor , @charlesh88 , @shefalijoshi , @nikhilmandlik , @JoelMcKinnon ,

I've set "vue/html-self-closing" as below. This means normal elements (div, span, etc.) must have separate opening and closing tags. Void elements like input do not use self-closing tags. ie. <input> is good but <input /> is not allowed. Components without inner content ARE allowed to be self-closed. svg and math are also allowed. This is okay per html5 spec.

Let me know if this is okay. If its too jarring I can just change this so that everything must have opening and closing tags (except voids).

            {
                "html": {
                    "void": "never",
                    "normal": "never",
                    "component": "always"
                },
                "svg": "always",
                "math": "always"
            }